src/api/core/Database.ts

```diff
@@ -1,15 +1,9 @@
 import { createConnection, Connection } from 'typeorm';
 import "reflect-metadata";
 
-class Database {
+export class Database {
```

Comment:
@bigLucas 
Since we don't intend to export anything else from the `database.ts` we can export as default, right?

Reply:
Yes, we can use the `default`, but it's a personal choice. Usually, I like to don't use the `default` keyword.